### DISCUSSION TITLE
Is abp.io with Blazorise free for commercial use?

### DISCUSSION THREAD

**Discussion by IbrahimTimimi** [abpframework/abp#14486] [General] [answered]
I wanted to remove Blazorise UI library from abp.io blazor project. I stumbled upon a post at [
How to remove Blazorise components #1902](https://support.abp.io/QA/Questions/1902/How-to-remove-Blazorise-components) by @maliming , which says:

> The abp core framework and modules are depending on Blazorise library. So I think you can't remove it at all.

The reason I wanted to remove Blazorise is because it is not free for commercial use as shown in [Pricing](https://commercial.blazorise.com/pricing). 

**Therefore, is abp.io which depends on Blazorise library free for commercial use or not?**

**Comment by hikalkan** [comment, marked as answer]
We made a deal with Blazorise's owner, so it is free for companies with **ABP Commercial** license. It is not possible to make it free for open source ABP users :)

**Comment by hikalkan** [comment]
BTW, Blazorise is free if your yearly income is below a certain amount. For more info, visit its own website.
